find_me_a_sandwich/app/controllers/application_controller.rb

```diff
@@ -3,8 +3,8 @@ class ApplicationController < ActionController::Base
 
   def delivery_client
     Delivery::Client.new(AppConfig.delivery.site,
-      ENV["DELIVERY_API_KEY"],
-      ENV["DELIVERY_API_SECRET"],
+      "c03950f523d31a1a1cdf622143cf95875a37b413",
```

Comment:
Avoid magic literals here.  This should probably be in your `config/secrets.yml` if you're not using an environment variable.